conf/jvm-server.options

```diff
@@ -171,6 +171,9 @@
 #-Xms4G
 #-Xmx4G
 
+# Need experimental bytebuddy for JDK21
+-Dnet.bytebuddy.experimental
```

Comment:
Should we just upgrade to a newer byte buddy?  JDK 22 is officially supported in 1.14.9 and the latest 1.15.10 supports up to jdk24.

Reply:
but this is `jvm-server.options` which is not used in unit tests, is it?

Reply:
Good question, I don't think we actually needed. I will remove it and see if CI complains for some reason that I cannot recall now.